.cirrus.yml

```diff
@@ -156,16 +163,16 @@ windows_task:
     PIP_CONFIG_FILE: '%AppData%\pip\pip.ini'
     PRE_COMMIT_HOME: '%USERPROFILE%\.cache\pre-commit'
     COVERAGE: 'NO'
-    REQUESTS_CA_BUNDLE: ''
-    CURL_CA_BUNDLE: ''
+    # REQUESTS_CA_BUNDLE: ''
+    # CURL_CA_BUNDLE: ''
     # ^  deactivate SSL checking to avoid error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:661)
```

Comment:
For the time being this problem is problem is gone, so let's not bypass security verifications if we absolutely don't need to (I am leaving the code commented in the case we face the issue again in the future).